hueniversitypy/theme_mcgill.py

```diff
@@ -16,7 +16,6 @@ def theme_mcgill():
 
     # Typography
```

Comment:
I would remove the original typography comment and make a comment that this is where the university font should be specified